src/components.jl

```diff
@@ -645,8 +645,8 @@ function (bc::BezierCurve)(t::Real)
     p
 end
 
-Base.mean(x::Real, y::Real) = 0.5*(x+y)
-Base.mean{N,T<:Real}(ps::FixedSizeArrays.Vec{N,T}...) = sum(ps) / length(ps)
+# mean(x::Real, y::Real) = 0.5*(x+y) #commented out as I cannot see this used anywhere and it overwrites a Base method with different functionality
```

Comment:
@tbreloff I am removing two functions here that 1. overwrite Base types, 2. are a little hassle to define NaN-compliant functions for, and 3. don't appear to be used anywhere in the code.